rakelib/link_transformer.rb

```diff
@@ -0,0 +1,22 @@
+# finds markdown files and tranforms markdown links into yard links
+class LinkTransformer
+
```

Comment:
I'm going to review style manually here for now. (We should probably set up rubocop to run on these classes, but I think we should that work to a separate PR because it might get large—there are likely other style issues extant in other rakelib files.)

For now:

* Style: No blank line between a class opener and the first method.